.github/workflows/tests.yml

```diff
@@ -135,12 +135,12 @@ jobs:
       - name: Adjust coverage file for Codecov
         run: bash <(sed -i 's/filename=\"/filename=\"pennylane\//g' coverage.xml)
 
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+      - uses: actions/upload-artifact@v2
         with:
-          file: ./coverage.xml
+          name: all-interfaces-coverage
+          path: ./coverage.xml
 
-  qcut:
+  qcut-tests:

```

Comment:
`qcut` and `qchem` are required checks to merge into `master`. Let me know if this PR is ready to be merged and will adjust the name of the checks. :slightly_smiling_face: